bundles/org.openhab.binding.electroluxair/README.md

```diff
@@ -0,0 +1,111 @@
+# ElectroluxAir Binding
+
+This is an openHAB binding for the Pure A9 Air Purifier, by Electrolux.
+
+This binding uses the Electrolux Delta REST API.
+
+![Electrolux Pure A9](doc/electrolux_pure_a9.webp)
+
+## Supported Things
+
+This binding supports the following types:
+
+- Bridge
+- Electrolux Pure A9
+
+If you define the bridge in a things-file the bridge type id is defined as `bridge`, e.g.:
+
+`Bridge electroluxair:api:myAPI "Electrolux Delta API" [username="user@password.com", password="12345", refresh="300"`
+
+If you define the thing in a things-file the thing type id is defined as:
+
+`Thing electroluxpurea9 MyElectroluxPureA9  "Electrolux Pure A9"    [ deviceId="123456789" ]`
```

Comment:
In this section you should only mention the ThingTypeUIDs and describe them shortly. The config examples could be moved to the "Full Examples" section.

Reply:
Removed!